src/main/scala/scynamo/wrapper/YearMonthFormatter.scala

```diff
@@ -0,0 +1,11 @@
+package scynamo.wrapper
```

Comment:
Hmm - what does `wrapper` stand for?

Reply:
This `package` already existed. I was not sure where to put this. I could also move it into `scynamo`. WDYT?

Reply:
I guess if we make it `private[scynamo]` it doesn't matter where it is.